Content.Server.Database/Migrations/Postgres/20200929113117_Init.cs

```diff
@@ -143,6 +143,27 @@ protected override void Up(MigrationBuilder migrationBuilder)
                         onDelete: ReferentialAction.Cascade);
                 });
 
+            migrationBuilder.CreateTable(
+                name: "trait",
+                columns: table => new
+                {
+                    trait_id = table.Column<int>(nullable: false)
+                        .Annotation("Npgsql:ValueGenerationStrategy", NpgsqlValueGenerationStrategy.IdentityByDefaultColumn),
+                    profile_id = table.Column<int>(nullable: false),
+                    trait_name = table.Column<string>(nullable: false),
+                    setting = table.Column<int>(nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_trait", x => x.trait_id);
+                    table.ForeignKey(
+                        name: "FK_trait_profile_profile_id",
+                        column: x => x.profile_id,
+                        principalTable: "profile",
+                        principalColumn: "profile_id",
+                        onDelete: ReferentialAction.Cascade);
+                });
+
```

Comment:
You should make a new migration for this rather than slotting it into the middle of initialisation...

Reply:
do i miss something on the docs.spacestation14.io?

Reply:
Is this the right comment? Not sure how this relates to the docs.